presto-main/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -403,4 +404,16 @@ public TaskManagerConfig setLegacySchedulingBehavior(boolean legacySchedulingBeh
         this.legacySchedulingBehavior = legacySchedulingBehavior;
         return this;
     }
+
+    public boolean isPagesIndexEagerCompact()
+    {
+        return pagesIndexEagerCompact;
+    }
+
+    @Config("task.pages-index.eager-compact")
```

Comment:
how about `pages-index.eager-compact-enabled` or `pages-index.eager-compaction-enabled`?

Reply:
I like `pages-index.eager-compaction-enabled` :). Will also use this for the variable name! :)